src/solver/conv_hip_implicit_gemm_grouped_wrw_xdlops.cpp

```diff
@@ -323,8 +323,6 @@ bool ConvHipImplicitGemmGroupWrwXdlops::IsApplicable(
         return false;
     if(miopen::IsEnabled(ENV(MIOPEN_DEBUG_CONVOLUTION_DETERMINISTIC)))
         return false;
-    if(problem.HasAtLeastOne64BitTensor())
-        return false;
```

Comment:
I am afraid we still need this because of https://github.com/ROCm/MIOpen/pull/2429/files#r1481674855

Reply:
It got renamed in `develop` and I started getting errors. I left the check with the new name `AllTensorsDimsFitIntoInt`.

Reply:
@amberhassaan Now I see.
[resolved]